static/templates/single_message.handlebars

```diff
@@ -6,32 +6,32 @@
-                             style="background-image: url('{{small_avatar_url}}');"/>
-                        {{#if status_message}}
-                            <span class="sender-status">
-                                <span class="sender_name-in-status sender_info_hover">{{msg/sender_full_name}}</span>
+                        {{#include_sender}}
+                            {{! See ../js/notifications.js for another user of avatar_url. }}
+                            <div class="u-{{msg/sender_id}} inline_profile_picture{{#status_message}} sender_info_hover{{/status_message}}"
+                                 style="background-image: url('{{small_avatar_url}}');"/>
+                            {{#if status_message}}
+                                <span class="sender-status">
+                                    <span class="sender_name-in-status sender_info_hover">{{msg/sender_full_name}}</span>
+                                    {{#if sender_is_bot}}
+                                    <img class="bot-icon" title="{{t 'Bot' }}" src="/static/images/bot.svg" />
+                                    {{/if}}
+                                    <span class="status-message">
+                                        {{{ status_message }}}
+                                    </span>
+                                    {{#if_and last_edit_timestr include_sender}}
+                                    <div class="message_edit_notice" title="{{#tr this}}Edited (__last_edit_timestr__){{/tr}}">{{t "EDITED" }}</div>
+                                    {{/if_and}}
+                                </span>
+                            {{else}}
+                                <span class="sender_name">{{msg/sender_full_name}}</span>
                                 {{#if sender_is_bot}}
                                 <img class="bot-icon" title="{{t 'Bot' }}" src="/static/images/bot.svg" />
                                 {{/if}}
-                                <span class="status-message">
-                                    {{{ status_message }}}
-                                </span>
-                                {{#if_and last_edit_timestr include_sender}}
-                                <div class="message_edit_notice" title="{{#tr this}}Edited (__last_edit_timestr__){{/tr}}">{{t "EDITED" }}</div>
-                                {{/if_and}}
-                            </span>
-                        {{else}}
-                            <span class="sender_name">{{msg/sender_full_name}}</span>
-                            {{#if sender_is_bot}}
-                            <img class="bot-icon" title="{{t 'Bot' }}" src="/static/images/bot.svg" />
                             {{/if}}
-                        {{/if}}
+                        {{/include_sender}}
                     </span>
-                    {{/include_sender}}
                     <span class="alert-copied pull-right">{{t 'Copied!' }}</span>
```

Comment:
I'm waiting on #4255 to get merged so I don't run into merge conflicts with the keyboard shortcuts documentation.